<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditForm</class>
 <widget class="QWidget" name="EditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>130</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>49</y>
     <width>695</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="edit_name">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Volkswagen polo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edit_year">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">max-width: 60px;</string>
      </property>
      <property name="text">
       <string>2004</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edit_milage">
      <property name="styleSheet">
       <string notr="true">max-width: 70px;</string>
      </property>
      <property name="text">
       <string>500000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="edit_body">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">max-width: 105px;</string>
      </property>
      <item>
       <property name="text">
        <string>Седан</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Купе</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Хэтчбек</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Универсал</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кроссовер</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Внедорожник</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Пикап</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Фургон</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Минивэн</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кабриолет</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="edit_gearbox">
      <property name="styleSheet">
       <string notr="true">max-width: 135px;</string>
      </property>
      <item>
       <property name="text">
        <string>Механическая</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Автоматическая</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Роботизированная</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="edit_drive">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">max-width: 55px;</string>
      </property>
      <item>
       <property name="text">
        <string>FWD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RWD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4WD</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="edit_position">
      <property name="styleSheet">
       <string notr="true">max-width: 35px;</string>
      </property>
      <item>
       <property name="text">
        <string>L</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>R</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_year">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Год  выпуска</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mileage">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Пробег</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gearbox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>25</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>КПП</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_drive">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>43</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Привод</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_body">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кузов</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_position">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>25</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Руль</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
